xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/service/BittrexAccountService.java

```diff
@@ -62,4 +69,23 @@ public AddressWithTag requestDepositAddressData(Currency currency, String... arg
   public TradeHistoryParams createFundingHistoryParams() {
     return TradeHistoryParamsZero.PARAMS_ZERO;
   }
+
+  @Override
+  public Map<Instrument, Fee> getDynamicTradingFeesByInstrument() throws IOException {
+    Map<CurrencyPair, Fee> dynamicTradingFees = getDynamicTradingFees();
+    return dynamicTradingFees.entrySet().stream().
+            collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
use proper error mapping as described in: https://github.com/knowm/XChange/wiki/New-Implementation-Best-Practices#step-7

Reply:
you can also see how error mapping should be done here in the methods above

Reply:
Error mapping corrected accordingly( as I read the wiki and also made committed code similar with other source code sections in Bittrex )